### PR TITLE
ReadPDFFile

### PR DESCRIPTION
**Pull request by JonathanMeler**
- Pase PDF File
- Extract Text, URLs, Metadata

### CODE REVIEW

File: Scripts/script-ReadPDFFile.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  required: true
+  default: true
+  description: War room entryID of the file to read.
+- name: maxFileSize
+  description: Maximal file size to load, in bytes. Default is 1MB.
+outputs:
+- contextPath: FileData
+  description: Data read from war room file
+scripttarget: 0
+dockerimage: orius123/pdfx
```
Comment (slavikm) [inline Scripts/script-ReadPDFFile.yml]: Please make sure to use demisto images

File: Scripts/script-ReadPDFFile.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          md += "\n* ".join(["{0}".format(str(k["Data"])) for k in URLs])
+
+          md += "\n### Text"
+          md += "\n{0}".format(text)
+
+          demisto.results({"Type" : entryTypes["note"],
+                          "ContentsFormat" : formats["markdown"],
+                          "Contents" : md,
+                          "HumanReadable": md,
+                          "EntryContext": {"File(val.EntryID = obj.EntryID)" : [pdfFile], "URL" : URLs}
```
Comment (slavikm) [inline Scripts/script-ReadPDFFile.yml]: I don't think you need an array for pdfFile, should just work with pdfFile, no?

File: Scripts/script-ReadPDFFile.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  description: War room entryID of the file to read.
+- name: maxFileSize
+  description: Maximal file size to load, in bytes. Default is 1MB.
+outputs:
+- contextPath: FileData
+  description: Data read from war room file
+scripttarget: 0
+dockerimage: orius123/pdfx
+outputs:
+- contextPath: URL
```
Comment (slavikm) [inline Scripts/script-ReadPDFFile.yml]: Should be URL.Data to be usable later as input

**Comment by JonathanMeler** [comment]
@slavikm changed to a demisto docker image

**Pull request merged**